orchestration/runner.go

```diff
@@ -74,10 +75,13 @@ func (r *Runner) runContainer(logger Logger, vd *variantData) error {
 	containerLinks := []string{}
 	for _, service := range servicesList {
 		name := fmt.Sprintf("service-%s-%s-%d", r.context.projectID, r.context.jobID, vd.variant.Number)
-		containerLinks = append(containerLinks, fmt.Sprintf("%s:%s", name, service))
+		if len(service.Alias) == 0 {
+			service.Alias = strings.Split(service.Image, ":")[0]
```

Comment:
Won't this break for qualified images ?
For example, if Image is `x/y`, we'll end up with `x/y` as Alias, which is not a valid value and will be rejected by docker.